src/main/java/com/google/api/codegen/DiscoveryFragmentGeneratorTool.java

```diff
@@ -67,18 +74,24 @@ public static void main(String[] args) throws Exception {
     generate(
         cl.getOptionValue("discovery_doc"),
         cl.getOptionValues("gapic_yaml"),
-        cl.getOptionValue("output", ""));
+        cl.getOptionValue("output", ""),
+        cl.getOptionValue("auth_instructions", ""));
   }
 
   @SuppressWarnings("unchecked")
   private static void generate(
-      String discoveryDoc, String[] generatorConfigs, String outputDirectory) throws Exception {
+      String discoveryDoc,
+      String[] generatorConfigs,
+      String outputDirectory,
+      String authInstructions)
+      throws Exception {
 
     ToolOptions options = ToolOptions.create();
     options.set(DiscoveryFragmentGeneratorApi.DISCOVERY_DOC, discoveryDoc);
     options.set(DiscoveryFragmentGeneratorApi.OUTPUT_FILE, outputDirectory);
     options.set(
         DiscoveryFragmentGeneratorApi.GENERATOR_CONFIG_FILES, Arrays.asList(generatorConfigs));
+    options.set(DiscoveryFragmentGeneratorApi.AUTH_INSTRUCTIONS_URL, authInstructions);
```

Comment:
Are you able to pass a list of config files? I can't get that array option to separate lists